flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Core/A32NX_ATSU.js

```diff
@@ -336,6 +340,19 @@ const uplinkRoute = async (mcdu) => {
                 await addWaypointAsync(fix, mcdu, fix.ident);
                 continue;
             }
+            if (fix.via_airway.match(/^NAT[A-Z]$/)) {
+                // NAT oceanic track
+                // MSFS has waypoints for these, but they're in a different format
+                console.log(`Inserting NAT waypoint: ${fix.ident}`);
+                try {
+                    const ident = simBriefToMsfsNatFix(fix);
```

Comment:
I also don't see the need of a function here, just use the return line of the defined arrow function in here instead since it has no other usage.

Or do we expect other usages for this in the near future?

Reply:
It didn't start life as a single line function is all. There is a readability argument even for single line functions but in this case I think it's well covered by the comment right above so I can inline it here.